### PR TITLE
update glow introduction

### PR DESCRIPTION
**Pull request by williambrandler**
Signed-off-by: wbrandler <william.brandler@databricks.com>

## What changes are proposed in this pull request?
(Details)

## How is this patch tested?
- [ ] Unit tests
- [ ] Integration tests
- [ ] Manual tests

(Details)

### CODE REVIEW

**Comment by williambrandler** [comment]
@henrydavidge updated the intro, please review when you have a moment

File: docs/source/introduction.rst [hunk 1]
```diff
@@ -1,23 +1,36 @@
 Introduction to Glow
 ====================
 
-Glow aims to simplify genomic workflows at scale. The best way to accomplish this goal is to take a system that
-has already been proven to work and adapt it to fit into the genomics ecosystem.
+Genomics data has been doubling every seven months globally. It has now reached a scale where genomics has
```
Comment (henrydavidge) [inline docs/source/introduction.rst:4]: Should it be genomic data or genomics data?

File: docs/source/introduction.rst [hunk 2]
```diff
@@ -1,23 +1,36 @@
 Introduction to Glow
 ====================
 
-Glow aims to simplify genomic workflows at scale. The best way to accomplish this goal is to take a system that
-has already been proven to work and adapt it to fit into the genomics ecosystem.
+Genomics data has been doubling every seven months globally. It has now reached a scale where genomics has 
+become a big data problem. However, most of the tools for working with genomics data are built to work on 
+single nodes and will no longer scale. Furthermore, it has become extremely challenging for scientists to
```
Comment (henrydavidge) [inline docs/source/introduction.rst:6]: drop extremely (adverbs suck)

File: docs/source/introduction.rst [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
-has already been proven to work and adapt it to fit into the genomics ecosystem.
+Genomics data has been doubling every seven months globally. It has now reached a scale where genomics has 
+become a big data problem. However, most of the tools for working with genomics data are built to work on 
+single nodes and will no longer scale. Furthermore, it has become extremely challenging for scientists to 
+manage storage of massive volumes of public data.
 
-Apache Spark and in particular `Spark SQL <https://spark.apache.org/sql/>`_, its module for working with
-structured data, is used at organizations across industries with datasets at the petabyte scale and
-beyond. Glow seamlessly integrates common bioinformatics tools into this big data ecosystem so that you can be productive immediately.
+Glow solves these problems by bridging bioinformatics into the big data ecosystem. The library enables bioinformaticians
```
Comment (henrydavidge) [inline docs/source/introduction.rst:9]: into => and

File: docs/source/introduction.rst [hunk 4]
```diff
@@ -1,23 +1,36 @@
...
+Genomics data has been doubling every seven months globally. It has now reached a scale where genomics has 
+become a big data problem. However, most of the tools for working with genomics data are built to work on 
+single nodes and will no longer scale. Furthermore, it has become extremely challenging for scientists to 
+manage storage of massive volumes of public data.
 
-Apache Spark and in particular `Spark SQL <https://spark.apache.org/sql/>`_, its module for working with
-structured data, is used at organizations across industries with datasets at the petabyte scale and
-beyond. Glow seamlessly integrates common bioinformatics tools into this big data ecosystem so that you can be productive immediately.
+Glow solves these problems by bridging bioinformatics into the big data ecosystem. The library enables bioinformaticians 
+and computational biologists to leverage best practices used by data engineers and data scientists across industry.
```
Comment (henrydavidge) [inline docs/source/introduction.rst:10]: industries?

File: docs/source/introduction.rst [hunk 5]
```diff
@@ -1,23 +1,36 @@
...
+- Ingest public genotype data into a data lake that acts as a single source of truth.
+- Run quality control, statistical analysis, and  association studies on population-scale datasets.
+- Build reproducible, production-grade genomics data pipelines that will scale to petabytes and beyond.
+
+![Glow architecture](/_static/images/glow_ref_arch_genomics.png)
 
 Glow features:
 
-- Genomic datasources: To read datasets in common file formats such as VCF and BGEN into Spark SQL DataFrames.
+- Genomic datasources: To read datasets in common file formats such as VCF, BGEN and Plink into Spark DataFrames.
```
Comment (henrydavidge) [inline docs/source/introduction.rst:26]: I'm surprised you'd leave out the oxford comma!

File: docs/source/introduction.rst [hunk 6]
```diff
@@ -1,23 +1,36 @@
...
 - Genomic functions: Common operations such as computing quality control statistics, running regression
-  tests, and performing simple transformations are provided as Spark SQL functions that can be
+  tests, and performing simple transformations are provided as Spark functions that can be
   called from Python, SQL, Scala, or R.
 - Data preparation building blocks: Glow includes transformations such as variant normalization and
   lift over to help produce analysis ready datasets.
-- Integration with existing tools: With Spark SQL, you can write user-defined functions (UDFs) in
-  Python, R, or Scala. Glow also makes it easy to run DataFrames through command line tools.
+- Integration with existing tools: With Spark, you can write user-defined functions (UDFs) in
```
Comment (henrydavidge) [inline docs/source/introduction.rst:32]: comma here too

File: docs/source/introduction.rst (hunk 1 above)
Comment (williambrandler) [reply docs/source/introduction.rst:4]: I've seen it used both ways

File: docs/source/introduction.rst (hunk 2 above)
Comment (williambrandler) [reply docs/source/introduction.rst:6]: fair

File: docs/source/introduction.rst (hunk 4 above)
Comment (williambrandler) [reply docs/source/introduction.rst:10]: across industry is used as a term for across all industries

**Comment by williambrandler** [comment]
thanks @henrydavidge, pushed changes

**Comment by henrydavidge** [comment]
Transient broken link. Merging.

**Pull request merged**